linkagetest.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
```

Comment:
remove this file. Instead, let's start a unit test folder. We did not have unit test before but we should have it now. Let's do that using Python's unittest module. Tests can be as simple as using `os.system` to run SEQLinkage without errors. But the test data should be small enough to run on a local machine and quickly produce results.

Reply:
removed